### PR TITLE
Enable use of Promotional Text and Promotional Codes.

### PR DESCRIPTION
**Pull request by AvBrocklehurst**
Added the ability to use promotional text and promotional codes in facebook app invites. 

Promotional Text and Promotional Codes can be used be adding the optional parameters promotionalText and promotionalCode to the appInvite object. 

```
{
                url: "http://examle.co.uk/applink",
                picture: "http://cdn.examle.co.uk/logo.png",
                promotionText: "Test Promotion",
                promotionCode: "Test Code"
},
```

### CODE REVIEW

**Comment by moniz1** [comment]
please merge this to the master branch

**Comment by moniz1** [comment]
in the ios section : promotionalCode is assigned for promotionalText please fix it

File: src/ios/FacebookConnectPlugin.m [hunk 1]
```diff
@@ -471,6 +474,13 @@ - (void) appInvite:(CDVInvokedUrlCommand *) command
         content.appInvitePreviewImageURL = [NSURL URLWithString:picture];
     }
 
+    if (promotionText && promotionCode) {
+        content.promotionCode = promotionText;
```
Comment (jeduan) [inline src/ios/FacebookConnectPlugin.m]: This is assigned backwards, as @moniz1 pointed out.

**Review by jeduan — COMMENTED** [review]
Seems good, thanks! Now, can you add docs to this @AvBrocklehurst? And sorry about the wait.

**Comment by AvBrocklehurst** [comment]
@jeduan Docs have been altered and the text and promo code switched around now.

**Comment by kimor29** [comment]
Can this be merged into master branch?

**Comment by kimor29** [comment]
Could this be enhanced with a getPromotionCode method?

**Comment by noahcooper** [comment]
This plugin is deprecated. Check out `cordova-plugin-facebook-connect` at https://www.npmjs.com/package/cordova-plugin-facebook-connect.

App Invites were deprecated by Facebook in 2018.

**Pull request closed without merging**